flit.ini

```diff
@@ -3,6 +3,7 @@ module=flit
 author=Thomas Kluyver
 author-email=thomas@kluyver.me.uk
 home-page=https://github.com/takluyver/flit
+license=LICENSE
```

Comment:
The license field is intended for the name or description of a license ([see metadata 1.2](https://www.python.org/dev/peps/pep-0345/#license-optional)), not a path.

I think I'd rather this was a convention-over-configuration thing, looking for a file called `LICENSE` adjacent to the module.